content/source/docs/enterprise/api/account.md

```diff
@@ -10,6 +10,83 @@ sidebar_current: "docs-enterprise2-api-account"
 
 Account represents the current user interacting with Terraform.
 
+## Update your account info
+
+Your username and email address can be updated with this endpoint.
+
+`PATCH /account/update`
+
+Status  | Response                                        | Reason
+--------|-------------------------------------------------|----------
+[200][] | [JSON API document][] (`type: "users"`)         | Your info was successfully updated
+[401][] | [JSON API error object][]                       | Unauthorized
+[422][] | [JSON API error object][]                       | Validation errors
+
+[200]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/200
+[401]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/401
+[422]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/422
+[JSON API document]: https://www.terraform.io/docs/enterprise/api/index.html#json-api-documents
+[JSON API error object]: http://jsonapi.org/format/#error-objects
+
+### Request Body
+
+This PATCH endpoint requires a JSON object with the following properties as a request payload.
+
+Key path                                   | Type    | Default  | Description
+-------------------------------------------|---------|----------|------------
+`data.type`                                | string  |          | Must be `"users"`
+`data.attributes.username`                 | string  |          | New username
+`data.attributes.email`                    | string  |          | New email address (must be confirmed afterwards to take effect)
```

Comment:
If you omit the .username or .email key, are they left as their previous values? This just came up in another area (workspaces PATCH), and after hemming and hawing I settled on showing the default value as `(previous value)`.

Reply:
@nfagerlund Yeah if username or email are omitted the attributes are not changed. For this endpoint you send what you want to change.